### DISCUSSION TITLE
[Question] Tabs: on_change event: How to retrieve the selected tab index?

### DISCUSSION THREAD

**Discussion by RobertNeher** [flet-dev/flet#1515] [Q&A] [answered]
### Question

How do I get the index of the selected tab in on_change trigger (selceted_index)?

### Code sample

```python
def onChange(e):
    print(selected_index)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by SKbarbon** [comment, marked as answer]
@RobertNeher ,

On your function:

selected_index = e.control.selected_index
